cirq-ionq/cirq_ionq/service.py

```diff
@@ -100,12 +101,15 @@ def run(
             seed: If the target is `simulation` the seed for generating results. If None, this
                 will be `np.random`, if an int, will be `np.random.RandomState(int)`, otherwise
                 must be a modulate similar to `np.random`.
+            error_mitigation: error mitigation settings
```

Comment:
This requires more explanation.  What are the keys and the values of this dictionary?